src/components/editTestCase/qdm/EditTestCase.tsx

```diff
@@ -244,11 +258,48 @@ const EditTestCase = () => {
           cqmMeasure,
           JSON.parse(currentTestCase?.json)
         );
+
+      //find the population_sets
+
+      var populationSets = JSONPath({
+        path: "$.population_sets[*].population_set_id",
+        json: cqmMeasure,
+      });
+
+      populationSets.forEach((pop) => {
+        var results = JSONPath({
+          path: `$..${pop}`,
+          json: calculationOutput,
+        });
+        let populationMap = new Map<String, number>();
+        let groupsMap = new Map<String, Map<String, number>>();
+
+        Object.entries(PopulationType).forEach((value, key) => {
+          //value is one of IPP, DENOM, NUMER, etc...
+          //Set's an entry = IPP & numeric value from results
+          populationMap.set(value[1], eval(`results[0].${value[0]}`));
+        });
+
+        groupsMap.set("" + pop, populationMap);
+
+        currentTestCase.groupPopulations.forEach((value) => {
+          if (value.groupId === pop) {
+            value.populationValues.forEach((population) => {
+              //console.log(`Group Population Values for ${value.groupId}`, JSON.stringify(population.name, null, 2));
```

Comment:
I think we need to delete this console.log

Reply:
Done